sbin/start-master.sh

```diff
@@ -31,7 +31,8 @@ if [[ "$@" = *--help ]] || [[ "$@" = *-h ]]; then
   echo "Usage: ./sbin/start-master.sh [options]"
   pattern="Usage:"
   pattern+="\|Using Spark's default log4j profile:"
-  pattern+="\|Registered signal handlers for"
+  pattern+="\|Started daemon with process name"
+  pattern+="\|Registered signal handler for"
```

Comment:
The output of `SignalUtils` is changed.

Reply:
Sorry, what is STS?

Reply:
Oh, I mean `Spark Thrift Server`.

Reply:
I tried but seems that STS has bigger problem other than such output: it didn't exist after parsing option `-h`. I suggest we can do it in another JIRA/PR.